docs/technical-reference.md

```diff
@@ -70,14 +104,9 @@ application load balancer
 - 2 private subnets for intra-VPC traffic to and from the Quilt RDS database and
 OpenSearch domain
 (1/8 of the VPC CIDR)
+- 2 user subnets (same subnet as for the load balancer)
```

Comment:
I don't think we create them - we only support them for customer-provided networks. (So I think the TF description got it backwards.) There's also no more space left - we already have 1/2, 1/4, 1/8, and 1/8 unused.